src/test/java/Homework21.java

```diff
@@ -0,0 +1,50 @@
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.devtools.v104.console.model.ConsoleMessage;
+import org.openqa.selenium.interactions.Actions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import org.openqa.selenium.WebDriver;
+
+import java.time.Duration;
+
+
+public class Homework21 extends BaseTest {
+
+    @Test(priority=1)
+    public void renamePlayList() throws InterruptedException {
+
+        provideEmail("iq14111991@gmail.com");
+        providePassword();
+        clickSubmitBtn();
+        renamePL();
+    }
+    private void renamePL() throws InterruptedException {
+
+        choseAndRenamePL();
+
+        verifyRenamedPL();
+    }
+
+    private void choseAndRenamePL(){
+        WebElement chosePL = driver.findElement(By.xpath("//a[contains(text(), 'CreatePLforRenaming')]"));
```

Comment:
![image](https://user-images.githubusercontent.com/104761482/202101127-e11bdd1e-0aae-4fa6-adb5-9c011faea6cb.png)


![image](https://user-images.githubusercontent.com/104761482/202100676-51af10e0-3ba6-4f54-8c2e-a6833a09699b.png)

Try like this:

![image](https://user-images.githubusercontent.com/104761482/202102071-c9ba1178-b9d9-4c92-b9b2-0d27a61b69cf.png)

```
  private void choseAndRenamePL(){
        WebElement chosePL = driver.findElement(By.xpath("//li[@class='playlist playlist'][1]"));
        Actions actions = new Actions(driver);
        actions.doubleClick(chosePL).perform();

        WebElement toClear = driver.findElement(By.name("name"));
        toClear.sendKeys(Keys.CONTROL + "a" + Keys.DELETE);
        toClear.sendKeys("It's WORKING????");
        toClear.sendKeys(Keys.ENTER );
    }
```